tests/helpers.py

```diff
@@ -27,52 +60,80 @@ def time_remaining(error):
 def decode_custom_error(contract_abi, error_data, w3):
     for error in [abi for abi in contract_abi if abi["type"] == "error"]:
         name = error["name"]
-        data_types = [collapse_if_tuple(abi_input) for abi_input in error.get("inputs", [])]
+        data_types = [
+            collapse_if_tuple(abi_input) for abi_input in error.get("inputs", [])
+        ]
         error_signature_hex = function_abi_to_4byte_selector(error).hex()
         if error_signature_hex.casefold() == str(error_data)[2:10].casefold():
-            params = ','.join([str(x) for x in w3.codec.decode(data_types,bytes.fromhex(str(error_data)[10:]))])
-            decoded = "%s(%s)" % (name , str(params))
+            params = ",".join(
+                [
+                    str(x)
+                    for x in w3.codec.decode(
+                        data_types, bytes.fromhex(str(error_data)[10:])
+                    )
+                ]
+            )
+            decoded = "%s(%s)" % (name, str(params))
```

Comment:
A little bit more modern:
```suggestion
            decoded = f"{name}({str(params)})"
```